lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -84,8 +84,8 @@ class S1ApUtil(object):
 
     _cond = threading.Condition()
     _msg = Queue()
-    # Default maximum wait time is 180 sec (3 min)
-    MAX_RESP_WAIT_TIME = 180
+    # Default maximum wait time is 60 sec (1 min)
+    MAX_RESP_WAIT_TIME = 60
```

Comment:
Are these changes needed as part of the fix?

Reply:
Removed the changes and will raise a separate PR for this